apps/frontend/src/app/5_pages/ErrorPage/ErrorPage.tsx

```diff
@@ -24,16 +27,18 @@ export const ErrorPage: FC = () => {
         <title>{t(translations.errorPage.meta.title)}</title>
       </Helmet>
       <Header />
-      <div className="container flex flex-col flex-grow items-center justify-center">
-        <h1>{t(translations.errorPage.title)}</h1>
-        <p>{t(translations.errorPage.description)}</p>
-        <p>
-          <i>{error.statusText || error.message}</i>
-        </p>
-        {process.env.NODE_ENV === 'development' && (
-          <pre>{JSON.stringify(error, undefined, 2)}</pre>
-        )}
-      </div>
+      <ErrorRenderer
+        error={
+          <>
+            <Paragraph className="w-full p-3">
+              <i>{error.statusText || error.message}</i>
+            </Paragraph>
+            {process.env.NODE_ENV === 'development' && (
```

Comment:
Extract the condition to a util method (e.g. `isTestnet`) and place it to `helpers.ts`.